examples/experimental/test/end_to_end/candid_rpc/candid_encoding/test/tests.ts

```diff
@@ -12,7 +12,7 @@ export function getTests(
     return () => {
         please('install didc', async () => {
             execSync(
-                `cargo install --git https://github.com/dfinity/candid --rev 5d3c7c35da652d145171bc071ac11c63d73bf803 --force didc`,
+                `cargo install --git https://github.com/dfinity/candid --rev 5d3c7c35da652d145171bc071ac11c63d73bf803 didc`,
```

Comment:
I guess I don't remember why we had --force but apparently we never needed or the need has gone away?